### ISSUE TITLE
[BUG] app crashes sometimes when we recover an Intent

### ISSUE THREAD

**Issue by mi-ka-git** [2018-SWENG/2018-SWENG-Radius#171]

**Comment by Oceanos1** [comment]
Next time i'll have this bug, I'll post the log from the console

**Comment by X4l1b1** [comment]
Maybe just a description of the situation in which it occurred would still help !

**Comment by Oceanos1** [comment]
Well, I tried to reproduce the issue but it seems to be a very difficult series of actions that end up in that crash..

I went to some different chats, and then I wanted to get back to HomeFragment with the Phone button and then, white screen -> app was crashed.

Sorry for the crappy description but I couldn't describe it later, I hope the bug occurred again before the deadline

**Comment by X4l1b1** [comment]
It seems to be an issue with the fragments Back stack. At some point Android tries to restore a null fragment or something when pressing on the "Back" Phone burton.

**Comment by Oceanos1** [comment]
Yes I remembered it was something like that but I can't figure how to reproduce the bug..
